.github/dependabot.yml

```diff
@@ -0,0 +1,24 @@
+version: 2
+updates:
+
+  # Maintain dependencies for GitHub Actions
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    open-pull-requests-limit: 5
+    labels: ["component/dependencies"]
+
+  # Maintain dependencies for dashboard, shell, and packages
+  - package-ecosystem: "npm"
+    directories:
+      - "/"
+      - "/shell"
+      - "/creators/extension"
```

Comment:
There's a fair few in this directory

- yarn creator's package.json
  - creators/extension/app/package.json
  - creators/extension/migrate/package.json
  - creators/extension/pkg/package.json
  - creators/extension/update/package.json
- package.json that will be used by developers in the content created by creator (app / extension)
  - creators/extension/app/app.package.json
  - creators/extension/pkg/pkg.package.json
- not quite sure on the root on creators/extension/package.json. must be associated with `init`

If there's no harm could all be checked?

Reply:
I don't believe that there will be any harm in doing so - I simply did not observe the nested nature of the creators directory. 

I added a line for globbing each nested `package.json` via `/creators/extension/*`, but I am unsure how we would support `app.package.json` and `pkg.package.json`. I don't see anything in the docs to suggest that we would be able to explicitly target these file patterns[^1]. If it comes down to it, we might need to manually bump the dependencies when the associated package.json gets updated.

[^1]: https://docs.github.com/en/code-security/dependabot/working-with-dependabot/dependabot-options-reference